Makefile

```diff
@@ -1473,6 +1477,21 @@ glance_kuttl: kuttl_common_prep glance glance_deploy_prep ## runs kuttl tests fo
 	make glance_cleanup
 	make kuttl_common_cleanup
 
+.PHONY: manila_kuttl_run
+manila_kuttl_run: ## runs kuttl tests for the manila operator,
+	kubectl-kuttl test --config ${MANILA_KUTTL_CONF} ${MANILA_KUTTL_DIR}
+
+.PHONY: manila_kuttl
+manila_kuttl: kuttl_common_prep ceph manila manila_deploy_prep ## runs kuttl tests for manila operator. Installs manila operator and cleans up previous deployments before and after running the tests.
+	$(eval $(call vars,$@,manila))
+	make wait
+	make manila_kuttl_run
+	make deploy_cleanup
+	make manila_cleanup
```

Comment:
nit: cleanup in the reverse order of install; `manila_cleanup` before `deploy_cleanup`